docs/examples/irradiance-transposition/plot_ghi_transposition.py

```diff
@@ -18,11 +18,10 @@
 from matplotlib import pyplot as plt
 
 # For this example, we will be using Golden, Colorado
-tz = 'MST'
 lat, lon = 39.755, -105.221
 
 # Create location object to store lat, lon, timezone
-site = location.Location(lat, lon, tz=tz)
+site = location.Location(lat, lon, tz='MST')
 
```

Comment:
I actually liked it the way it was before when `tz` was a variable, defined together with lat and lon where it's easy to comment. IMHO moving it here makes the code more terse, less verbose, and less accessible. In examples especially, I think it's better to lean towards over-explaining and writing as clearly and verbosely as possible.

Reply:
I agree, I like the pattern of having configurations in a group at the top of the script so that people don't have to hunt through the code to make sure they didn't miss something if they want to model a different location.  

Also if we get rid of the variable for `tz`, why not do the same for `lat` and `lon`?